mne_bids/meg_bids.py

```diff
@@ -41,9 +44,11 @@ def _mkdir_p(path):
 def _channel_tsv(raw, fname, verbose):
     """Create channel tsv."""
 
-    map_chs = dict(grad='MEGGRAD', mag='MEGMAG', stim='TRIG', eeg='EEG',
+    map_chs = defaultdict(lambda: 'OTHER')
+    map_chs.update(grad='MEGGRAD', mag='MEGMAG', stim='TRIG', eeg='EEG',
                    eog='EOG', ecg='ECG', misc='MISC', ref_meg='REFMEG')
-    map_desc = dict(grad='Gradiometer', mag='Magnetometer',
+    map_desc = defaultdict(lambda: 'Other type of channel')
```

Comment:
The spec has a restricted list of channel types but I wonder if there is a way to preserve some info we extract to put in the description. e.g. `resp` is a channel type in the BTi data but that info would be lost

Reply:
hmm ... you can always add it in the description, but it's true that "other" is no longer an available option. Perhaps we should update the MEG draft for that? Although I'd rather not have the draft updated at this point so we can write the software (validator etc.) with what we have